engine/src/test/java/io/camunda/zeebe/engine/processing/timer/TimerCatchEventTest.java

```diff
@@ -204,8 +204,6 @@ public void shouldTriggerTimer() {
             .withProcessInstanceKey(processInstanceKey)
             .getFirst();
 
-    ENGINE.increaseTime(Duration.ofSeconds(1));
-
```

Comment:
@remcowesterhoud Are you sure we should also delete the time increase? The time increase made the issue clear, but it still seems reasonable. I'd suggest increasing the time by 0.1s though.

Reply:
I think the problem with that is that it will remain flaky that way. Sometimes the engine time is still increased before the timer is scheduled. I'm not entirely sure why but if I increase the engine time by 100ms I can't get it to succeed.

Reply:
Hmm what I said doesn't make sense. But the assertion will fail because the timer is now triggered after 200ms instead 100ms. I will take a luncbreak, maybe my subconscious will explain it to me 😄

Reply:
@korthout I reckon it has to do with the `TIMER_RESOLUTION` of 100ms. This would cause the test to fail if we increase the time since we check that the timer gets triggered within 100ms ± 50ms.

We don't really need to increase the time here though. The timer gets triggered within 100ms. This won't slow down the tests enough to be significant. I would propose to leave it out.
I will change the assertion to check that it has been triggered within 100-150ms. The current assertion would also pass if it gets triggered at 50ms, which is not intended.

Reply:
Thanks for looking into it and clarifying @remcowesterhoud 

Let's keep it without the time traveling 👍